File: Задание 4.py
import math
vvod=int(input())
rad=vvod*math.pi/180
print(round(rad, 4))
print(round(math.sin(rad), 4))
print(round(math.cos(rad), 4))
print(round(math.sin(rad)/math.cos(rad), 4))
print(round(math.cos(rad)/math.sin(rad), 4))

File: Code/First_work/Задание 4.pas
var vvod, rad:real;
begin
  readln(vvod);
  rad:=vvod*Pi/180;
  writeln('радианы ',rad:0:4);
  writeln('синус ',sin(rad):0:4);
  writeln('косинус ',cos(rad):0:4);
  writeln('тангенс ',sin(rad)/cos(rad):0:4);
  writeln('котангенс ',cos(rad)/sin(rad):0:4);
end.
